manifests/role/kubernetes/controller.pp

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2019 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# Kubernetes controller plane and etcd server
+class nebula::role::kubernetes::controller {
+  include nebula::profile::base
+  include nebula::profile::work_around_puppet_bugs
+  include nebula::profile::apt
+  include nebula::profile::authorized_keys
+  include nebula::profile::vim
+  include nebula::profile::kubernetes::firewall
```

Comment:
All these lines should just be `include nebula::role::minimal`, but I can't do that until the base firewall supports ignoring some chains.